modules/sftp-transfer/main.tf

```diff
@@ -0,0 +1,38 @@
+locals {
```

Comment:
add some outputs to [outputs.tf](https://github.com/cloudposse/terraform-aws-components/pull/394/files#diff-de3fdcdbbf5d238c2a8e8342746e0ef9f44d888b72c06e58d93d4b47084a83e6)